test/namespaces.bats

```diff
@@ -32,3 +34,47 @@ function teardown() {
 	output=$(crictl exec --sync "$ctr_id" cat /proc/1/cmdline)
 	[[ "$output" == *"pause"* ]]
 }
+
+@test "pid namespace mode target test" {
+	CONTAINER_NAMESPACES_DIR="$namespaces_dir" start_crio
+
+	pod1="$TESTDIR"/sandbox1.json
+	jq '	  .linux.security_context.namespace_options = {
+			pid: 1,
+		}' \
+		"$TESTDATA"/sandbox_config.json > "$pod1"
+	ctr1="$TESTDIR"/ctr1.json
+	jq '	  .linux.security_context.namespace_options = {
+			pid: 1,
+		}' \
+		"$TESTDATA"/container_redis.json > "$ctr1"
+
+	target_ctr=$(crictl run "$ctr1" "$pod1")
+
+	pod2="$TESTDIR"/sandbox2.json
+	jq --arg target "$target_ctr" \
+		'	  .linux.security_context.namespace_options = {
+			pid: 3,
+			target_id: $target
+		}
+		| .metadata.name = "sandbox2" ' \
+		"$TESTDATA"/sandbox_config.json > "$pod2"
+
+	ctr2="$TESTDIR"/ctr2.json
+	jq --arg target "$target_ctr" \
+		'	  .linux.security_context.namespace_options = {
+			pid: 3,
+			target_id: $target
+		}' \
+		"$TESTDATA"/container_sleep.json > "$ctr2"
+
+	ctr_id=$(crictl run "$ctr2" "$pod2")
+
+	output1=$(crictl exec --sync "$target_ctr" ps | grep -v ps)
+	output2=$(crictl exec --sync "$ctr_id" ps | grep -v ps)
+	[[ "$output1" == "$output2" ]]
+
+	crictl rmp -fa
+	# make sure namespace is cleaned up
+	[[ -z $(ls "$namespaces_dir/pidns") ]]
```

Comment:
Perhaps it's more clear to check that the entry does not exist? Something like `[ ! -e "$namespaces_dir/pidns" ]`